onnxruntime/contrib_ops/cpu/bert/attention.cc

```diff
@@ -227,30 +227,16 @@ Status Attention<T>::PrePack(const Tensor& weights, int input_idx, bool& is_pack
 template <typename T>
 Status Attention<T>::Compute(OpKernelContext* context) const {
   const Tensor* input = context->Input<Tensor>(0);
-  const Tensor* weights;
```

Comment:
I thought I've fixed the warnings in this file, and you are reverting my changes back?